### DISCUSSION TITLE
配置文件怎么配置啊？有更加详细的流程吗？大佬们

### DISCUSSION THREAD

**Discussion by yuweuss** [mikumifa/biliTickerBuy#266] [Ideas]
![image](https://github.com/mikumifa/biliTickerBuy/assets/130210786/0c5886fd-90c6-4ea6-a351-2838b7766f36)
可以运行
点击没有反应，是要配置什么文件吗？
![image](https://github.com/mikumifa/biliTickerBuy/assets/130210786/f39a2643-74b9-4d12-8555-1526a2436c7b)

**Comment by Asuna380** [comment]
获取票信息啊，有使用说明书的

**Reply by yuweuss** [reply]
说明书看了啊，是要自己写配置文件吗？说明书没写啊

**Reply by Asuna380** [reply]
你获取票信息啊，选好之后就能直接生成配置文件啊

**Reply by yuweuss** [reply]
老铁为什么点击后没有反应，终端就那两行，是要买的票的时间问题吗？改了要抢票的时间也没用啊

**Reply by peigenr** [reply]
点了过一会儿会弹出一个浏览器让你登录的，如果没有请检查你的任务栏和电脑配置（

**Reply by yuweuss** [reply]
> 点了过一会儿会弹出一个浏览器让你登录的，如果没有请检查你的任务栏和电脑配置（

压根没启动webdriver啊，也不显示报错，就没反应

**Reply by zunduwuyu** [reply]
注意关闭一些防火软件，比如火绒，我之前也不弹，关了就弹了

**Reply by yuweuss** [reply]
> 注意关闭一些防火软件，比如火绒，我之前也不弹，关了就弹了
F12检查一看
Unable to preload CSS for https://gradio.s3-us-west-2.amazonaws.com/assets/index-CAbm2BVw.css

**Reply by yuweuss** [reply]
> 注意关闭一些防火软件，比如火绒，我之前也不弹，关了就弹了
解决了，不用克隆项目，直接用博主打包的项目，同时把自己电脑的防火墙关了就行了

**Reply by yuweuss** [reply]
有点奇怪，一会儿可以用，一会又用不了了（

**Comment by zdjjsofii** [comment]
![Uploading IMG_9709.jpeg…]()
大佬们完成验证码怎么点了没用啊

**Comment by mikumifa** [comment]
要先登录，会弹出一个浏览器

**Reply by yuweuss** [reply]
> 要先登录，会弹出一个浏览器

有点奇怪了，是加载资源速度慢了还是，防火墙关了，又可以了。会弹出浏览器登录的bushi(，后面信息就存在json文件里了，感谢博主
